quickstart/azure/setup.md

```diff
@@ -56,11 +56,12 @@ Service Principal is the recommended way to connect Pulumi to Azure in a team or
 
 Once obtained, there are two ways to communicate your authorization tokens to Pulumi:
 
-1. Set the environment variables `ARM_CLIENT_ID`, `ARM_CLIENT_SECRET`, and `ARM_TENANT_ID`, respectively
+1. Set the environment variables `ARM_CLIENT_ID`, `ARM_CLIENT_SECRET`, `ARM_TENANT_ID`, and `ARM_SUBSCRIPTION_ID` respectively
 
 2. Set them using configuration
 
     ```bash
+    $ pulumi config set azure:subscriptionId <subscriptionId>
```

Comment:
It'd be nice if this was moved to the end of the commands, to match the order of the environment variables above.